src/evo/providertx.h

```diff
@@ -23,7 +23,7 @@ class CValidationState;
 
 struct ErrReasonAndMessage {
     ValidationInvalidReason reason;
-    std::string_view message;
```

Comment:
why don't you like string_view? It doesn't own string but do not require for string to be C-style null-ended